ajax/libs/alertify/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "alertify",
+  "description": "An unobtrusive customizable JavaScript notification system",
+  "version": "0.3.11",
+  "homepage": "https://fabien-d.github.com/alertify.js/",
```

Comment:
`https` seems to not work on this site, let's use `http`.

Reply:
@maruilian11 It is working for me

Reply:
Works for me too
@maruilian11 maybe try again?